docs/sources/shared/visualizations/axis-options-state-status.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Axis options
+comments: |
+  This file is used in the following visualizations: state timeline, status history.
+---
+
+| Option    | Description                                                                                      |
+| --------- | ------------------------------------------------------------------------------------------------ |
+| Placement | Controls visibility of series names along the Y axis or time values along the x axis             |
```

Comment:
Sorry @imatwawana , we're gonna need another round on this with a new attribute we added